components/resources/wallet_strings.grdp

```diff
@@ -132,6 +132,17 @@
   <message name="IDS_BRAVE_WALLET_OTHERS" desc="Swap max slippage others label">Others: <ph name="SLIPPAGE_PERCENT"><ex>1</ex>$1</ph>%</message>
   <message name="IDS_BRAVE_WALLET_SUGGESTED" desc="Swap max slippage suggested option">Suggested</message>
   <message name="IDS_BRAVE_WALLET_CUSTOM" desc="Swap max slippage custom option">Custom</message>
+  <message name="IDS_BRAVE_WALLET_CHEAPEST" desc="Swap route option cheapest label">Cheapest</message>
+  <message name="IDS_BRAVE_WALLET_FASTEST" desc="Swap route option fastest label">Fastest</message>
+  <message name="IDS_BRAVE_WALLET_ON_NETWORK" desc="Swap token on a specific network">on <ph name="NETWORK_NAME"><ex>Ethereum Mainnet</ex>$1</ph></message>
```

Comment:
Are these "Cheapest on ..." and "Fastest on ..."? If so, it might be better to just have 2 full strings, so that the l10lizers have better context. These are short strings so the word count won't jump much.

Reply:
Is just a `Logo` and then `on NETWORK_NAME`

![Screenshot 29](https://github.com/user-attachments/assets/f58832e7-260f-47f0-a220-2d3ad81a10ed)